components/leaflet-maps/ClustersMap.vue

```diff
@@ -195,7 +195,53 @@
        })
 
        return optionalsWithValue
-     }
+     },
+     getFormatDate: function (date,precision){
+
+      let [day, month, year] = date.split('/');
+      day = Number(day);
+      month = Number(month)
+      year = Number(year);
+      let currentDate = new Date(year, month - 1, day);
+      
+      
+      //the precision of the dates can either be or 9 11(full day/month/year),10(month/year),9(year)
+      
+      // If we don't know the day, month, and year, then wikidata precision will be less than 11 and the dates don't need Julian conversion
+      if (precision === '10') {
+          // Then return the date with month and year only
+          let dateString = `${(currentDate.getMonth() + 1).toString().padStart(2, '0')}/` + `${currentDate.getFullYear()}`;
```

Comment:
the variable `dateString` can be removed here. You could change the code to just to an automatic return e.g.

```
return `${(currentDate.getMonth() + 1).toString().padStart(2, '0')}/` + `${currentDate.getFullYear()}`;
```

Same can be applied to the return further down

Reply:
Changed. Thank you